### PR TITLE
feat: Add container registry agent support

### PR DESCRIPTION
**Pull request by jurglic**
Add broker support for container-registry-agent by providing:
* template: accept.json, env
* Dockerfile to build image
* README section

- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/broker/blob/master/.github/CONTRIBUTING.md) rules

### CODE REVIEW

File: client-templates/container-registry-agent/accept.json.sample [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "origin": "${CR_AGENT_URL}"
+  },
+  {
+    "method": "GET",
+    "path": "/get-state-identifier",
+    "origin": "${CR_AGENT_URL}"
+  },
+  {
+    "method": "GET",
+    "path": "/list*",
```
Comment (snaftaly) [inline client-templates/container-registry-agent/accept.json.sample:34]: Looking at other broker client examples, I think you can split this into two entries, one with the path: "/list" (for listing repos) and another one with the path "/list/:repo" (for listing tags).

File: client-templates/container-registry-agent/accept.json.sample [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  {
+    "method": "POST",
+    "path": "/api/v2/import/result"
+  }
+  ],
+  "private": [
+  {
+    "method": "GET",
+    "path": "/healthcheck",
```
Comment (snaftaly) [inline client-templates/container-registry-agent/accept.json.sample:19]: I'm not sure this is needed, as the registry will not send healthchecks to the remote DRA. wdyt?

File: client-templates/container-registry-agent/accept.json.sample [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (snaftaly) [inline client-templates/container-registry-agent/accept.json.sample:1]: A general comment - consider adding a "//" comment field in each endpoint, to explain its purpose.

File: client-templates/container-registry-agent/accept.json.sample (hunk 2 above)
Comment (jurglic) [reply client-templates/container-registry-agent/accept.json.sample:19]: Not strictly necessary atm. But imho good to have this open from the get go, if we want to monitor it in the future.

File: client-templates/container-registry-agent/accept.json.sample (hunk 1 above)
Comment (jurglic) [reply client-templates/container-registry-agent/accept.json.sample]: Updated

File: client-templates/container-registry-agent/accept.json.sample (hunk 3 above)
Comment (jurglic) [reply client-templates/container-registry-agent/accept.json.sample:1]: Done

**Review by iamtmrobinson — APPROVED** [review]

**Pull request merged**